programs/launcher/main.cpp

```diff
@@ -1110,24 +1110,27 @@ launcher_def::launch (eosd_def &instance, string &gts) {
   node_rt_info info;
   info.remote = !host->is_local();
 
-  string eosdcmd = "programs/eosd/eosd ";
+  string eosdcmd(eosd_binary);
+
   if (skip_transaction_signatures) {
-    eosdcmd += "--skip-transaction-signatures ";
+    eosdcmd += " --skip-transaction-signatures";
```

Comment:
There are other command line options that also need this change.  Per the Travis build, at least `--plugin` needs to be similarly updated.  There may be others.